src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -74,7 +76,12 @@ Real64 BenchmarkHotgym::run(UInt EPOCHS, bool useSPlocal, bool useSPglobal, bool
 
   // initialize SP, TP, Anomaly, AnomalyLikelihood
   tInit.start();
-  ScalarEncoder enc(133, -100.0, 100.0, DIM_INPUT, 0.0, 0.0, false);
+  ScalarEncoderParameters encParams;
+  encParams.active = 133;
```

Comment:
nit, call this param activeBits for clarity?

Reply:
Done.  Good advice!